Python/Bar Code Generator

```diff
@@ -0,0 +1,9 @@
+# Generating Barcodes in Python
```

Comment:
That's a pretty well-documented code. If possible, also please do either mention in the code file as a comment, or in a separate file the working of this code.

Reply:
Yeah, for sure